CONTRIBUTING.md

```diff
@@ -77,10 +77,18 @@ discussion at [discuss.emberjs.com](http://discuss.emberjs.com)
 Building Ember is quite simple.
 
 ```sh
-cd ember.js
-npm install
-bower install
-npm run-script build
+clone the latest ember.js directory from github
+ - git clone https://github.com/emberjs/ember.js.git
+
+follow these commands to build ember.js
+ - sudo su
```

Comment:
Running all this code as an elevated user isn't something we feel is responsible to recommend. This code should be run at no more then a regular users privs